pkg/server/httpserver/auth.go

```diff
@@ -64,66 +70,20 @@ func (s *httpServer) Run(handler func()) (err error) {
 
 		tlsConfig := &tls.Config{
 			ClientCAs:  caCertPool,
-			ClientAuth: tls.RequestClientCert,
+			ClientAuth: tls.RequireAndVerifyClientCert,
```

Comment:
Does using this change the status code for missing certificates and invalid certificates?

Reply:
@shivanshs9 Yes, it seems that if tls certs verify failed, it will return with http status code `500` directly.

Reply:
@iguoyr Is that fine though? To have a backwards incompatible 500 error for it? Maybe we can keep the previous functions to verify certificates.

Reply:
Well, I think you're right! Already rollback the code :)